### ISSUE TITLE
[Feature] An Admin should be able to do User management

### ISSUE THREAD

**Issue by ylorgat** [faloker/purify#119]
**Is your feature request related to a problem? Please describe.**
When you've assigned an Admin to a project they should have the ability to mange Users assigned to that project as well,

For example Reset passwords for those Users (this could be other admins on the project with them, Users and observers)

**Describe the solution you'd like**

When an admin clicks manage project -> manage users they have the ability to request a new password just for the Users in the same project as them. Not just add and remove them from the project. 

<img width="133" alt="Screenshot 2020-12-09 at 11 59 03" src="https://user-images.githubusercontent.com/35497024/101627233-f25f7b00-3a15-11eb-8f13-6e5e1b819851.png">

Thank you very much :)

**Comment by faloker** [comment]
I spend quite a bit of time thinking and experimenting to implement the behavior described, but it turns out to be a poor fit. 

I think the main problem is the option to reset the password. I don't like this option because it essentially allows you to take over another user account. But this was done because I can't require SMTP for email delivery, and most organizations use SSO, so they don't need this feature.

Simply put, if this feature request is implemented, administrators can hijack each other's and other users' accounts and do other things that would be a complete mess. So I would prefer it to be as simple as it is now - only owners can manage users, including resetting passwords.

**Comment by ylorgat** [comment]
Hey there,

Thank you very much for your comment,

What about creating groups and assigning users to a group and each has "group"
